contracts/dao/schemes/UBI.sol

```diff
@@ -186,15 +188,16 @@ contract UBI is AbstractUBI {
     )
         public
         AbstractUBI(_avatar, _identity, _initialReserve, _periodStart, _periodEnd)
-    {}
+    {
+        claimers = (identity.whitelistedCount()).sub(identity.whitelistedContracts());
```

Comment:
this will cause formula not to be dynamic in number of claimers

Reply:
Yes, but only for the type of ubi contract where one can only claim if they've been whitelisted before the start of the contract. This is why the amount of potential claimers are calculated at the start.